### DISCUSSION TITLE
RTP packet with unknown payload type: 50.

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#4035] [General]
Hello Team MTX,

Thank you for the excellent product.

During testing with a white-labeled Univew dome camera (IPC3618SB-ADF28KMC-I0), Univew XVR (XVR302-16Q3) with analog cameras, and the Uniarx XVR (Uniarch XVR 108G3) and Uniarx analog cameras, we encountered the error "[RTSP source] received RTP packet with unknown payload type: 50." Payload Type 50 specifically refers to the H.263 codec in RTP contexts, which aligns with the RTP/AVP (Audio-Visual Profile), where payload type 50 is commonly used for H.263 video. However, when I checked the settings on these XVRs and cameras, the video compression was set to H.264. We also updated the firmware, but the issue persists.

Please investigate this issue and provide guidance on how to resolve it. I appreciate your support and assistance.

**Comment by aler9** [comment]
moved to #4036
